docs/BulkImport.md

```diff
@@ -15,8 +15,8 @@ The Bulk import feature enables importing FHIR data in the NDJSON format to the
 
 ### Current limitations
 
-* Conditional references in resources are not supported for initial mode import.
-* If multiple resources share the same resource ID, then only one of those resources will be imported at random and an error will be logged corresponding to the remaining resources sharing the ID.
+* In initial mode, resources with conditional references will be logged as errors. In incremental mode, resources with conditional references will be loaded but references will not be resolved.
+* If multiple resources share the same resource ID, in initial mode, only one of those resources will be imported at random and an error will be logged corresponding to the remaining resources sharing the ID. In incremental mode, if lastUpdated is not provided, behavior is the same as in the initial. If lastUpdated is provided, then resource versions are deduplicated by lastUpdated, and only duplicates are reported as errors, the rest is loaded.
```

Comment:
If lastUpdated is provided, then resource versions are deduplicated by lastUpdated, and only duplicates are reported as errors, the rest is loaded. --- is this true with allowed version flag ? In case there are multiple resources with same version id and different lastUpdated, then error will be reported?

Reply:
1. This is "universally" true irrespective of allow negative versions flag.
2. If there are multiple records for the same resource with different lastUpdated and the same version, then version conflict will be reported.

Reply:
For statement "In incremental mode, if lastUpdated is not provided, behavior is the same as in the initial."  -  Initial mode import creates resources and not updates. So if lastUpdated is not provided but incremental version ids are provided, then the resource is updated.  

Should we update the statement?

Reply:
"If there are multiple records for the same resource with different lastUpdated and the same version, then version conflict will be reported." - Version conflict will be reported on the randomly selected resource ?

Reply:
The answer is "not predictable" record. The reason is that answer is dependent on order of processing across batches, and only in a single batch any order can be guaranteed.

Reply:
> For statement "In incremental mode, if lastUpdated is not provided, behavior is the same as in the initial." - Initial mode import creates resources and not updates. So if lastUpdated is not provided but incremental version ids are provided, then the resource is updated.
> 
> Should we update the statement?

Input versions are ignored if lastUpdated is not provided. This has existed since day one for incremental mode. There is no need to update the statement.